addons/binding/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/handler/EchoHandler.java

```diff
@@ -722,6 +724,8 @@ public void updateState(AccountHandler accountHandler, @Nullable Device device,
             @Nullable JsonNotificationSound @Nullable [] alarmSounds,
             @Nullable List<JsonMusicProvider> musicProviders) {
         try {
+            this.logger.debug("Handle updateState {}", this.getThing().getUID().getAsString());
```

Comment:
You do not have to call the `getAsString()` method for logging the UID. The logger takes care of that itself.

Reply:
Thank you for this hint, I will change this in my next PR